connexion/decorators/produces.py

```diff
@@ -101,15 +182,21 @@ def __call__(self, function):
         def wrapper(*args, **kwargs):
             url = flask.request.url
             data, status_code, headers = self.get_full_response(function(*args, **kwargs))
+            logger.debug((data, status_code, headers))
```

Comment:
Apologies, this was some testing code of mine that ended up in the commit. I've removed for now.